app/models/email.py

```diff
@@ -0,0 +1,36 @@
+import uuid
+from datetime import datetime
+
+from django.db import models
+from enumfields import Enum, EnumField
+
+from .game import Game
+
+class EmailRule(Enum):
```

Comment:
I really don't like the naming here. `EmailRule` implies that there's a rule about the email, not the the specified recipient group. I think something like `RecipientGroup` or `RecipientCode` would be better here.